spec/models/dish_spec.rb

```diff
@@ -9,6 +9,23 @@
 
   describe "relationships" do
       it {should belong_to :chef}
+      it { should have_many :dish_ingredients}
```

Comment:
Nice work testing relationships in this model method! MAke sure you do this with your other models as well! There should be relationship tests in the chef_spec, ingredient_spec and dish_ingredient_spec as well